app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<section><h1>Recipes</h1>
+  <h3><%=@recipe.name %></h3>
+  <p>Complexity: <%= @recipe.complexity %></p>
+  <p>Genre: <%= @recipe.genre %></p>
+</section>
+
+<section>
+  <h4>Ingredients</h4>
+  <p><% @ingredients.each do |ingredient| %></p>
```

Comment:
This is where I would've just done `@recipe.ingredients.each...` instead of neededing a second instance variable.